ideascube/mediacenter/tests/test_views.py

```diff
@@ -223,30 +223,53 @@ def test_tags_link_should_update_querystring(app):
         response.request.GET.dict_of_lists()
 
 
-def test_remove_filter_should_be_present(app):
+def test_remove_filter_should_be_present(app, catalog):
+    from ideascube.serveradmin.catalog import Package
+    # We use the available_packages in the view to get the display name
+    # of package in the remove filter link.
+    # But to have some available_packages, we need content that match the search.
+    DocumentFactory(lang='en', title='foo', tags=['tag1', 'tag2'], kind='video', package_id='test.package1')
+
+    # Also add the test.package1 in the catalog.
+    catalog.add_mocked_package(Package('test.package1', {
+        'name':'Test package1',
+        'description':'Test package1 description',
+        'template_id':'template_name',
+        'language':'fr',
+        'staff_only':False}))
+
     response = app.get(reverse('mediacenter:index'),
                        {'lang': 'en',
                         'q': 'foo',
                         'tags': ['tag1', 'tag2'],
-                        'kind': 'video'})
+                        'kind': 'video',
+                        'source': 'test.package1'})
 
     links = response.pyquery('.filters.card a').filter(
         lambda i, elem: "English" in (elem.text or ''))
     assert links
     resp = app.get("{}{}".format(reverse('mediacenter:index'),
                                  links[0].attrib['href']),
                    status=200)
-    assert {'q': ['foo'], 'tags': ['tag1', 'tag2'], 'kind': ['video']} == \
-        resp.request.GET.dict_of_lists()
+    assert resp.request.GET.dict_of_lists() == {
+        'q': ['foo'],
+        'tags': ['tag1', 'tag2'],
+        'kind': ['video'],
+        'source': ['test.package1']}
```

Comment:
Isn't that just checking that Django did its job right? Shouldn't we just trust them to populate `request.GET` correctly?